runtimes/pythonrt/py-sdk/autokitteh/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""AutoKitteh Python SDK."""
+
+from . import atlassian
+from .attr_dict import AttrDict
+from .decorators import activity
+from . import errors
+from . import google
```

Comment:
Don't import `google` and `slack`. It'll force users to have the corresponding SDK installed.